data/shell-completions/zsh/_meson

```diff
@@ -217,7 +217,7 @@ local -a meson_commands=(
     "$__meson_cd"
     '--no-rebuild[Do not rebuild before installing]'
     '--only-changed[Do not overwrite files that are older than the copied file]'
-    '--quiet[Do not print every file that was installed]'
+    '(--quiet -q)'{'--quiet','-q'}'[Do not print every file that was installed]'
```

Comment:
Please add this to the bash completion as well.

Reply:
Thanks, forgot that!

Reply:
I think you added this to the wrong commit?

Reply:
Oh, damn will fix.

Reply:
Fixed!